dev-util/android-sdk-update-manager/android-sdk-update-manager-26.1.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+PYTHON_COMPAT=(python3_{10..11})
+
+inherit python-r1 udev distutils-r1
+
+DESCRIPTION="Open Handset Alliance's Android SDK"
+HOMEPAGE="https://developer.android.com"
+SRC_URI="https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip"
+RESTRICT="mirror"
+
+LICENSE="android"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-arch/tar
+		app-arch/unzip
+		app-arch/gzip
+		acct-group/android"
+
+RDEPEND=">=virtual/jdk-1.5
+	>=dev-java/ant-core-1.6.5
+	dev-java/swt:3.7[cairo]
+	sys-libs/ncurses-compat:5[abi_x86_32(-)]
+	sys-libs/zlib[abi_x86_32(-)]
+"
+
+ANDROID_SDK_DIR="/opt/${PN}"
+QA_FLAGS_IGNORED_x86="
+	${ANDROID_SDK_DIR/\//}/tools/adb
+	${ANDROID_SDK_DIR/\//}/tools/mksdcard
+	${ANDROID_SDK_DIR/\//}/tools/sqlite3
+	${ANDROID_SDK_DIR/\//}/tools/hprof-conv
+	${ANDROID_SDK_DIR/\//}/tools/zipalign
+	${ANDROID_SDK_DIR/\//}/tools/dmtracedump
+"
+QA_FLAGS_IGNORED_amd64="${QA_FLAGS_IGNORED_x86}"
+
+QA_PREBUILT="${ANDROID_SDK_DIR/\//}/tools/*"
+
+S="${WORKDIR}/tools"
+
+src_compile() {
+    :
+}
+
+src_prepare() {
+    rm -rf tools/lib/x86*
```

Comment:
Please don't use the f. Use `rm -r tools/lib/x86* || die "Failed to remove tools/lib/x86*`